packages/api-reference/src/components/Content/Content.vue

```diff
@@ -66,9 +73,40 @@ const introCardsSlot = computed(() =>
         </div>
       </template>
     </Introduction>
+
+    <!-- Show servers/clients even if we have no intro -->
+    <SectionContainer v-else-if="servers || baseServerURL">
+      <Section class="introduction-section">
+        <SectionContent>
+          <SectionColumns>
+            <SectionColumn>
+              <div
+                class="introduction-card"
+                :class="{ 'introduction-card-row': layout === 'accordion' }">
+                <BaseUrl
+                  class="introduction-card-item"
+                  :defaultServerUrl="baseServerURL"
+                  :servers="props.servers"
+                  :specification="parsedSpec" />
+                <Authentication
+                  class="introduction-card-item"
+                  :parsedSpec="parsedSpec"
+                  :proxy="proxy" />
+                <ClientLibraries class="introduction-card-item" />
+              </div>
+            </SectionColumn>
```

Comment:
I think if you just add an empty section column after this then it will restrict it to the left hand column and match the endpoint descriptions.

Reply:
but then we got an empty void on the left. Which is better?

Reply:
Empty void on the right is what I'm suggestion, keep the block left just reduce width

Reply:
ooo yes brilliant will do 🫡